### PR TITLE
DEVELOPER.md: add link to vscode docs

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Long Dai <long0dai@foxmail.com>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/main/PULL_REQUESTS.md)

Commit Message:

It is hard for new developers to find how to setup vscode env, so add it.

Additional Description:
Risk Level: Low
Testing:
Docs Changes: Yes
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: DEVELOPER.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ To generate a coverage report, there is a [coverage build script](https://github
 
 If you plan to contribute to Envoy, you may find it useful to install the Envoy [development support toolchain](https://github.com/envoyproxy/envoy/blob/main/support/README.md), which helps automate parts of the development process, particularly those involving code review.
 
+If you want to setup local development environment with Visual Studio Code, please follow with [vscode setup](https://github.com/envoyproxy/envoy/blob/main/tools/vscode/README.md).
```
Comment (phlax) [inline DEVELOPER.md:12]: "if you want to set up a local development..."

Comment (daixiang0) [reply DEVELOPER.md]: Updated.

File: DEVELOPER.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ To generate a coverage report, there is a [coverage build script](https://github
 
 If you plan to contribute to Envoy, you may find it useful to install the Envoy [development support toolchain](https://github.com/envoyproxy/envoy/blob/main/support/README.md), which helps automate parts of the development process, particularly those involving code review.
 
+If you want to set up a local development environment with Visual Studio Code, please follow with [vscode setup](https://github.com/envoyproxy/envoy/blob/main/tools/vscode/README.md).
```
Comment (phlax) [inline DEVELOPER.md:12]: "please follow with" sounds a bit wrong - how about changing it to just "please see"

Comment (daixiang0) [reply DEVELOPER.md:12]: OK

**Comment by daixiang0** [comment]
@phlax could it be merged?

**Review by phlax — APPROVED** [review]
lgtm, thanks @daixiang0

**Review by phlax — CHANGES_REQUESTED** [review]
hmm - actually @daixiang0 im looking at the document overall and thinking that this should maybe be in the list below

**Comment by daixiang0** [comment]
I update it, please review again.

**Review by phlax — APPROVED** [review]
lgtm - thanks for updating @daixiang0

**Pull request merged**